### PR TITLE
feat: [AFM] new Product in APIM

### PR DESCRIPTION
**Pull request by jacopocarlini**
### List of changes

Added a new product in the APIM: "AFM calculator for Node"

### Motivation and context

We want expose a subset of the APIs to Nodo dei pagamenti with subscription required

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

**Review by MarcoAGreco — DISMISSED** [review]

File: src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    - Remove the <base> element to prevent inheriting policies from the corresponding section element in the enclosing scope.
+    - Policies are applied in the order of their appearance, from the top down.
+    - Comments within policy elements are not supported and may disappear. Place your comments between policy elements or at a higher level scope.
+-->
+<policies>
+  <inbound>
+    <base />
+    <set-backend-service base-url="https://${hostname}/pagopa-afm-calculator-service" />
+
+    <set-variable name="isMock" value="@(context.Request.Headers.GetValueOrDefault("Ocp-Apim-Subscription-Key","").Equals("{{afm-dev-secondary-sub-key}}") || true)" />
```
Comment (aomegax) [inline src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml:18]: remove `true`

File: src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    <choose>
+      <when condition="@(context.Variables.GetValueOrDefault<bool>("isMock"))">
+      <!--Mock Responses-->
+      <set-variable name="statusCode" value="@(context.Request.Headers.GetValueOrDefault("Ocp-Apim-Error-code", "200"))" />
```
Comment (aomegax) [inline src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml:22]: `<set-variable name="statusCode" value="@(context.Request.Headers.GetValueOrDefault("Ocp-Apim-Error-Code", "200"))" />`

File: src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+          "idBundle": "1",
+          "idCiBundle": "1",
+          "idPsp": "88888888888"
+          }
+          ]</set-body>
+      </return-response>
+    </when>
+    </choose>
+    <!--Scenario 3-->
+    <choose>
```
Comment (aomegax) [inline src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml:80]: This is _scenario 4_
_Scenario 3_ is missing

File: src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        <set-status code="@(int.Parse((string)context.Variables["statusCode"]))" />
+        <set-header name="Content-Type" exists-action="override">
+          <value>application/json</value>
+        </set-header>
+        <set-body>[
+          {
+          "taxPayerFee": 80,
+          "primaryCiIncurredFee": 0,
+          "paymentMethod": "PO",
+          "touchpoint": null,
```
Comment (aomegax) [inline src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml:61]: I think touchpoint should have `ANY` value instead of `null`

**Review by aomegax — CHANGES_REQUESTED** [review]
great work! well done! 🥇

File: src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml (hunk 4 above)
Comment (jacopocarlini) [reply src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml]: fixed

File: src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml (hunk 2 above)
Comment (jacopocarlini) [reply src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml]: fixed

File: src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml (hunk 1 above)
Comment (jacopocarlini) [reply src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml]: done

File: src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml (hunk 3 above)
Comment (jacopocarlini) [reply src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml:80]: [scenario 3](https://pagopa.atlassian.net/wiki/spaces/PPR/pages/550634053/Mock#response.2) has empty list as response

**Review by aomegax — APPROVED** [review]
👍

**Review by pasqualespica — DISMISSED** [review]
great 👍

**Pull request merged**